runner/src/server/types.ts

```diff
@@ -31,7 +31,7 @@ type Services = (
   uploadService: UploadService;
   webhookService: WebhookService;
   statusService: StatusService;
-  queueService: QueueService;
+  queueService: MySqlQueueService;
```

Comment:
This should still be QueueService right? The new abstract class

Reply:
yep, good catch